specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2018-05-01/componentProactiveDetection_API.json

```diff
@@ -0,0 +1,315 @@
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource Id"
+        },
+        "name": {
+          "type": "string",
+          "description": "Azure resource name"
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource type"
+        },
+        "location": {
+          "type": "string",
+          "description": "Resource location"
+        },
+        "properties": {
+          "description": "Properties that define a ProactiveDetection configuration.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ApplicationInsightsComponentProactiveDetectionConfigurationProperties"
+        }
+      }
+    },
+    "ApplicationInsightsComponentProactiveDetectionConfigurationProperties": {
+      "description": "Properties that define a ProactiveDetection configuration.",
+      "type": "object",
+      "properties": {
+        "Name": {
+          "type": "string",
+          "readOnly": false,
+          "description": "The rule name"
+        },
+        "Enabled": {
+          "type": "boolean",
+          "readOnly": false,
+          "description": "A flag that indicates whether this rule is enabled by the user"
+        },
+        "SendEmailsToSubscriptionOwners": {
+          "type": "boolean",
+          "readOnly": false,
+          "description":
+            "A flag that indicated whether notifications on this rule should be sent to subscription owners"
+        },
+        "CustomEmails": {
+          "type": "array",
+          "readOnly": false,
+          "description": "Custom email addresses for this rule notifications",
+          "items": {
+            "type": "string"
+          }
+        },
+        "LastUpdatedTime": {
+          "type": "string",
+          "readOnly": false,
+          "description": "The last time this rule was updated"
+        },
+        "RuleDefinitions": {
+          "type": "object",
+          "readOnly": false,
+          "description":
+            "Static definitions of the ProactiveDetection configuration rule (same values for all components).",
+          "properties": {
+            "Name": {
+              "type": "string",
+              "readOnly": false,
+              "description": "The rule name"
+            },
+            "DisplayName": {
+              "type": "string",
+              "readOnly": false,
+              "description": "The rule name as it is displayed in UI"
+            },
+            "Description": {
+              "type": "string",
+              "readOnly": false,
+              "description": "The rule description"
+            },
+            "HelpUrl": {
+              "type": "string",
+              "readOnly": false,
+              "description": "URL which displays aditional info about the proactive detection rule"
```

Comment:
```suggestion
              "description": "URL which displays additional info about the proactive detection rule"
```

Reply:
done.

---
In reply to: [270709310](https://github.com/Azure/azure-rest-api-specs/pull/5518#discussion_r270709310) [](ancestors = 270709310)